frontend/src/pages/manga/[id].tsx

```diff
@@ -80,7 +80,7 @@ const MangaHeader = ({manga}: { manga: MangaResponse }) => {
 			<div className={"relative p-5 z-10 flex flex-1"}>
 				<div className={"flex flex-1 space-x-4"}>
 					<Image
-						src={manga.metadata.coverImageUrl}
+						src={manga.metadata.coverImageUrl! /* TODO: Fallback image */}
```

Comment:
Build was failing because of a change in the model in another commit. The `src` cannot be null, so we should have a fallback URL here.